lib/insights/api/common/paginated_response.rb

```diff
@@ -2,6 +2,8 @@ module Insights
   module API
     module Common
       class PaginatedResponse
+        ASSOCIATION_COUNT_ATTR = "@count".freeze
```

Comment:
This should probably be something more generic to contain a list of operators.  I could see adding things like `@min` and `@max` in the future.